client/migrations/sql/cockroach/1.sql

```diff
@@ -0,0 +1,39 @@
+-- +migrate Up
```

Comment:
How can we ensure that each of those migration files is up to date with the latest table from the other migrations?

Reply:
Aren't the tests for that? 😉. I migrated to the latest both a postgres and a cockroach database and verified the columns one by one, I'll do another pass now we stabilized the change a little bit.